### PR TITLE
various: replace guava Files usages JDK Files

### PR DESCRIPTION
**Pull request by jobegrabber**
Motivation:

From #3808: The dCache heavily utilizes Google's guava library. For some
use cases the JDK standard library now offers similar functionality.
Here a few  com.google.common.io.Files can be replaced with java.nio.file.Files
invocations.

Modification:

Reduce dependency on external library guava by utilizing
java.nio.file.Files' methods instead of the ones provided by
com.google.common.io.Files (where applicable).

Result:

Keeping behaviour the same while having reduced dependency on guava.

Target: master
Signed-off-by: Jonas Grabber <jobegrabber@gmail.com>

### CODE REVIEW

**Comment by kofemann** [comment]
ok to test

**Comment by jobegrabber** [comment]
I'm not sure how to read the failing tests, for example [this one](https://ci.dcache.org/job/GridTools-Functional-Tests/2533/robot/DccpTests%20&%20GeneralFileCopyTests%20&%20GlobusurlcpTests%20&%20SrmcpTests%20srmV2%20&%20SrmlsTests%20srmV2%20&%20SrmmvTests%20srmV2%20&%20SrmsetpermTests%20srmV2%20&%20SrmspaceTests%20srmV2/GeneralFileCopyTests/GLOBUS/):
`error: globus_ftp_client: the server responded with an error 451 No write pools online for [net=131.169.240.54,protocol=GFtp/2,store=<Unknown>:<Unknown>@osm,cache=,linkgroup=]`

Looks like this could be an issue due to the changes I applied, though I'm not sure. Could you confirm, @kofemann ?

**Comment by kofemann** [comment]
retest this please

**Comment by kofemann** [comment]
@jobegrabber looks unrelated to me. But I will have a deeper look.

File: modules/dcache/src/main/java/org/dcache/services/billing/text/Indexer.java [hunk 1]
```diff
@@ -25,24 +24,10 @@
...
-import java.io.ObjectInputStream;
-import java.io.ObjectOutputStream;
-import java.io.OutputStream;
-import java.io.OutputStreamWriter;
-import java.io.PipedReader;
-import java.io.PipedWriter;
-import java.io.PrintStream;
-import java.io.PrintWriter;
-import java.io.Reader;
+import java.io.*;
```
Comment (kofemann) [inline modules/dcache/src/main/java/org/dcache/services/billing/text/Indexer.java]: usually we don't use star imports.

**Comment by kofemann** [comment]
Thanks. Looks good to me.

**Comment by jobegrabber** [comment]
@kofemann expanded the star import in `Indexer` to single class imports.

**Comment by kofemann** [comment]
Thanks!

**Pull request merged**